201-asev2-ilb-with-web-app/azuredeploy.json

```diff
@@ -0,0 +1,157 @@
+      "metadata": {
+        "description": "Defines which worker pool's (WP1, WP2 or WP3) resources will be used for the app service plan."
+      }
+    },
+    "numberOfWorkersFromWorkerPool": {
+      "type": "int",
+      "defaultValue": 1,
+      "metadata": {
+        "description": "Defines the number of workers from the worker pool that will be used by the app service plan."
+      }
+    }
+  },
+  "resources": [
+    {
+      "apiVersion": "2016-09-01",
+      "type": "Microsoft.Web/hostingEnvironments",
+      "name": "[parameters('aseName')]",
+      "kind": "ASEV2",
+      "location": "[parameters('aseLocation')]",
+      "tags": {
+        "displayName": "ASE Environment",
+        "usage": "Hosting PaaS applications",
+        "category": "Environment",
+        "owner": "[parameters('owner')]"
+      },
+      "properties": {
+        "name": "[parameters('aseName')]",
+        "location": "[parameters('aseLocation')]",
+        "ipSslAddressCount": 0,
+        "internalLoadBalancingMode": "[parameters('internalLoadBalancingMode')]",
+        "dnsSuffix": "[parameters('dnsSuffix')]",
+        "virtualNetwork": {
+          "Id": "[parameters('vnetResourceName')]",
+          "Subnet": "[parameters('subnetName')]"
+        }
+      }
+    },
+    {
+      "apiVersion": "2016-09-01",
+      "name": "[parameters('appServicePlanName')]",
+      "type": "Microsoft.Web/serverfarms",
+      "location": "[parameters('aseLocation')]",
+      "tags": {
+        "displayName": "ASE Hosting Plan",
+        "usage": "Hosting Plan within ASE",
+        "category": "Hosting",
+        "owner": "[parameters('owner')]"
+      },
+      "properties": {
+        "name": "[parameters('appServicePlanName')]",
+        "hostingEnvironmentProfile": {
+          "id": "[resourceId('Microsoft.Web/hostingEnvironments/',parameters('aseName'))]"
+        }
+      },
+      "sku": {
+        "name": "[concat('I',parameters('workerPool'))]",
+        "tier": "Isolated",
+        "size": "[concat('I',parameters('workerPool'))]",
+        "family": "I",
+        "capacity": "[parameters('numberOfWorkersFromWorkerPool')]"
+      },
+      "dependsOn": [
+        "[concat('Microsoft.Web/hostingEnvironments/',parameters('aseName'))]"
+      ]
+    },
+    {
+      "apiVersion": "2016-08-01",
+      "name": "[parameters('siteName')]",
+      "type": "Microsoft.Web/sites",
+      "location": "[resourceGroup().location]",
+      "tags": {
+        "displayName": "ASE Web App",
+        "usage": "Web App Hosted within ASE",
+        "category": "Web App",
+        "owner": "[parameters('owner')]"
+      },
+      "properties": {
+        "name": "[parameters('siteName')]",
+        "serverFarmId": "[resourceId('Microsoft.Web/serverFarms',parameters('appServicePlanName'))]",
+        "hostingEnvironmentProfile": {
+          "id": "[resourceId('Microsoft.Web/hostingEnvironments/', parameters('aseName'))]"
+        }
+      },
+      "dependsOn": [
```

Comment:
move this above properties

Reply:
What do you mean with move? For as far as I know this property should be there.

Reply:
done